package.json

```diff
@@ -18,16 +18,20 @@
     "eslint-config-airbnb-base": "12.1.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-import": "^2.13.0",
+    "eth-contract-class": "^0.0.12",
```

Comment:
A few of these packages don't seem to be used anywhere

Reply:
theyre used in the submodule

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/2063b3b3-2054-4a07-83b6-0a10a04de771-f765aaee-8456-4925-a425-f49e3c63b510-v1.png?transparent=1&palette=1&width=246)